components/invitation/InviteViaLink.tsx

```diff
@@ -83,9 +84,10 @@ const InviteViaLink = ({ team }: InviteViaLinkProps) => {
       }
     );
 
+    const result = (await response.json()) as ApiResponse;
```

Comment:
You can keep it inside the if condition in all the places, it is saving unnecessary read of the body.